style/rails/sample.rb

```diff
@@ -2,4 +2,20 @@ class SomeClass
   belongs_to :tree
   has_many :apples
   has_many :watermelons
+
+  def self.with_apples
+    includes(:apples)
+  end
+
+  def self.with_watermelons
+    includes(:watermelons)
+  end
+
+  def best_fruit
+    "avocado"
+  end
+
+  def fruit
+    ["banana", "grape", "zebra"]
```

Comment:
Well, I started off with primary colors, so a bit longer than if I had taken the more obvious route of searching for secondary colored zebras...